vault/identity_store_entities.go

```diff
@@ -800,6 +862,28 @@ func (i *IdentityStore) mergeEntity(ctx context.Context, txn *memdb.Txn, toEntit
 		}
 
 		for _, alias := range fromEntity.Aliases {
+			if aliasId, ok := toEntityAccessors[alias.MountAccessor]; ok {
```

Comment:
Could you use fromAlias and toAliasID as variable names for improved readability?

Reply:
Done!